spring-cloud-gcp-secretmanager/src/main/java/org/springframework/cloud/gcp/secretmanager/SecretManagerTemplate.java

```diff
@@ -40,6 +40,11 @@
  */
 public class SecretManagerTemplate implements SecretManagerOperations {
 
+	/**
+	 * Default value for the latest version of the secret.
+	 */
+	public static final String latestVersion = "latest";
```

Comment:
Oh yup, I see. We would prefer all uppercase name here: `LATEST_VERSION`.

Reply:
Pushed